src/main/resources/db/changesets/20240604_renameupdateddateonrequest.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: rename_updated_date_on_snapshot_request
+      author: rjohanek
+      changes:
+        - renameColumn:
+            tableName: snapshot_request
+            oldColumnName: updated_date
+            newColumnName: status_updated_date
```

Comment:
yes, the only time we update this field is when the status is updated. When we update the snapshot access request to include the flight id or the created snapshot id, we do not update this field. This is because we think the user will care most about when the request is approved/rejected.